features-json/variable-fonts.json

```diff
@@ -0,0 +1,318 @@
+{
+  "title":"Variable fonts",
+  "description":"Font format that supports",
+  "spec":"https://drafts.csswg.org/css-fonts-4/",
+  "status":"wd",
+  "links":[
+    {
+      "url":"https://bugs.webkit.org/show_bug.cgi?id=162815",
+      "title":"Webkit status"
+    },
+    {
+      "url":"https://bugs.chromium.org/p/chromium/issues/detail?id=669453",
+      "title":"Chromium status"
+    },
+    {
+      "url":"https://bugzilla.mozilla.org/show_bug.cgi?id=1302685",
+      "title":"Firefox status"
+    }
+  ],
+  "bugs":[
+
+  ],
+  "categories":[
+    "CSS",
+    "JS API"
+  ],
+  "stats":{
+    "ie":{
+      "5.5":"n",
+      "6":"n",
+      "7":"n",
+      "8":"n",
+      "9":"n",
+      "10":"n",
+      "11":"n"
+    },
+    "edge":{
+      "12":"n",
+      "13":"n",
+      "14":"n",
+      "15":"n",
+      "16":"n"
+    },
+    "firefox":{
+      "2":"n",
+      "3":"n",
+      "3.5":"n",
+      "3.6":"n",
+      "4":"n",
+      "5":"n",
+      "6":"n",
+      "7":"n",
+      "8":"n",
+      "9":"n",
+      "10":"n",
+      "11":"n",
+      "12":"n",
+      "13":"n",
+      "14":"n",
+      "15":"n",
+      "16":"n",
+      "17":"n",
+      "18":"n",
+      "19":"n",
+      "20":"n",
+      "21":"n",
+      "22":"n",
+      "23":"n",
+      "24":"n",
+      "25":"n",
+      "26":"n",
+      "27":"n",
+      "28":"n",
+      "29":"n",
+      "30":"n",
+      "31":"n",
+      "32":"n",
+      "33":"n",
+      "34":"n",
+      "35":"n",
+      "36":"n",
+      "37":"n",
+      "38":"n",
+      "39":"n",
+      "40":"n",
+      "41":"n",
+      "42":"n",
+      "43":"n",
+      "44":"n",
+      "45":"n",
+      "46":"n",
+      "47":"n",
+      "48":"n",
+      "49":"n",
+      "50":"n",
+      "51":"n",
+      "52":"n",
+      "53":"n",
+      "54":"n",
+      "55":"n",
+      "56":"n",
+      "57":"n",
+      "58":"n",
+      "59":"n"
+    },
+    "chrome":{
+      "4":"n",
+      "5":"n",
+      "6":"n",
+      "7":"n",
+      "8":"n",
+      "9":"n",
+      "10":"n",
+      "11":"n",
+      "12":"n",
+      "13":"n",
+      "14":"n",
+      "15":"n",
+      "16":"n",
+      "17":"n",
+      "18":"n",
+      "19":"n",
+      "20":"n",
+      "21":"n",
+      "22":"n",
+      "23":"n",
+      "24":"n",
+      "25":"n",
+      "26":"n",
+      "27":"n",
+      "28":"n",
+      "29":"n",
+      "30":"n",
+      "31":"n",
+      "32":"n",
+      "33":"n",
+      "34":"n",
+      "35":"n",
+      "36":"n",
+      "37":"n",
+      "38":"n",
+      "39":"n",
+      "40":"n",
+      "41":"n",
+      "42":"n",
+      "43":"n",
+      "44":"n",
+      "45":"n",
+      "46":"n",
+      "47":"n",
+      "48":"n",
+      "49":"n",
+      "50":"n",
+      "51":"n",
+      "52":"n",
+      "53":"n",
+      "54":"n",
+      "55":"n",
+      "56":"n",
+      "57":"n",
+      "58":"y x #1",
+      "59":"y x #1",
+      "60":"y x #1",
+      "61":"y x #1",
```

Comment:
If the feature's only supported behind a flag, the value should be `n d #1`. Also the `x` refers to requiring a prefix, is that correct here?

Reply:
Thanks @Fyrd & @thlinard - I've addressed all of the above comments (Fyrd it is indeed a flag and not prefixing, so I've updated all the relevant entries). Let me know if there's anything more I need to do/fix!